common/lib/capa/capa/templates/optioninput.html

```diff
@@ -1,7 +1,7 @@
 <% doinline = "inline" if inline else "" %>
 
 <form class="inputtype option-input ${doinline}">
-   <select name="input_${id}" id="input_${id}" aria-label="${label}" aria-describedby="answer_${id}">
+   <select name="input_${id}" id="input_${id}" aria-label="${response_data['label']}" aria-describedby="answer_${id} ${response_data['description_ids']}">
```

Comment:
@cahrens `aria-label="${response_data['label']}"` will be removed in [TNL-4970](https://openedx.atlassian.net/browse/TNL-4970). Didn't removed in this PR to avoid test failures.